cmd/up/exec.go

```diff
@@ -214,6 +214,10 @@ func (eg *envsGetter) getEnvs(ctx context.Context) ([]string, error) {
 
 	envs = append(envs, eg.getDefaultLocalEnvs()...)
```

Comment:
when i tested this last week the envs from the field `environment` where not appearing at the dev container.... have we check this is working?

Reply:
1. Since the command is executed locally I don't think we need those values in the dev container. Also when doing the `exec` cmd we get them in the command so I think we cover the cases that require them